js/app/components/player/provider.tsx

```diff
@@ -1,62 +1,20 @@
 // basically PlayerProvider that sets up our magic context,
 
-import { LivestreamProvider } from "@streamplace/components";
-import { newPlayer, PlayerContext } from "features/player/playerSlice";
-import { useContext, useEffect, useState } from "react";
-import { ReadyState } from "react-use-websocket";
-import { useAppDispatch } from "store/hooks";
+import {
+  LivestreamProvider,
+  PlayerProvider as NewPlayerProvider,
+} from "@streamplace/components";
 import { PlayerProps } from "./props";
 
-const POLL_INTERVAL = 3000;
 // PlayerInner starts doing player stuff
 export default function PlayerProvider(
   props: Partial<PlayerProps> & { children: React.ReactNode },
 ): React.ReactNode {
-  const ctx = useContext(PlayerContext);
-  if (ctx.playerId) {
-    return props.children;
-  }
   return (
-    <LivestreamProvider src={props.src ?? ""}>
-      <PlayerContextInitializer {...props}>
+    <NewPlayerProvider>
+      <LivestreamProvider src={props.src ?? ""}>
         {props.children}
-      </PlayerContextInitializer>
-    </LivestreamProvider>
+      </LivestreamProvider>
+    </NewPlayerProvider>
   );
 }
```

Comment:
Potentially overkill to have a separate file for this now, could just wrap the `player.tsx` export in a `<LivestreamProvider />` and be happy